### PR TITLE
Add progress option

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

This PR is for describing a training progress.

## This PR includes

- Set default `EARLY_STOPPING_ROUNDS = 10`
- Remove "feature_fraction_bynode"
- Add training progress option
- Fix for warning

## What type of PR is it?

Feature

## What is the issue?

N/A

## How should this be tested?

N/A

### CODE REVIEW

File: callbacks.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from logging import getLogger
```
Comment (keigohtr) [inline callbacks.py]: Same as the other's callbacks.

File: callbacks.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        statistics.add_stage(ABEJAStatistics.STAGE_VALIDATION,
+                             val_acc, val_loss)
+
+        try:
+            self.client.update_statistics(statistics)
+        except Exception:
+            logger.warning('failed to update statistics.')
+
+
+class TensorBoardCallback(object):
```
Comment (keigohtr) [inline callbacks.py]: This callback is for lightgbm

File: parameters.py [hunk 3]
```diff
@@ -46,7 +46,10 @@ def get_env_var_metric(key, list_):
     for value in values:
         if value not in list_:
             raise Exception(f'"{key}" must be one of [{",".join(list_)}]')
-    return ",".join(values)
+    rtn = ",".join(values)
+    if rtn:
+        return rtn
+    return None
```
Comment (keigohtr) [inline parameters.py]: Official doc allows us to use `metric=""` but it warns when running.
microsoft/LightGBM#2285

File: train.py [hunk 4]
```diff
@@ -106,7 +112,8 @@ def handler(context):
     dtrain = lgb.Dataset(X_train, y_train)
     
     extraction_cb = ModelExtractionCallback()
-    callbacks = [extraction_cb,]
+    tensorboard_cb = TensorBoardCallback(statistics, writer)
+    callbacks = [extraction_cb, tensorboard_cb,]
```
Comment (ysku) [inline train.py]: [nits] redundant `,`

Comment (keigohtr) [reply train.py]: 👍

**Comment by keigohtr** [comment]
Thank you for reviewing!

**Pull request merged**